dev-libs/boxfort/boxfort-0.0.1_pre20200902.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit meson python-any-r1
+
+BOXFORT_COMMIT="ac0507b3f45fe58100b528baeb8ca04270b4a8ff"
+
+DESCRIPTION="Convenient & cross-platform sandboxing C library"
+HOMEPAGE="https://github.com/Snaipe/BoxFort"
+SRC_URI="https://github.com/Snaipe/BoxFort/archive/${BOXFORT_COMMIT}.tar.gz -> boxfort-${BOXFORT_COMMIT}.tar.gz"
```

Comment:
You know I'm going to say I prefer -> ${P}.tar.gz ;)

Reply:
Ok, changed :)